lib/src/models/data_item.dart

```diff
@@ -49,13 +62,18 @@ class DataItem implements TransactionBase {
     List<Tag>? tags,
     String? data,
     Uint8List? dataBytes,
+    String? dataSize,
   })  : target = target ?? '',
         nonce = nonce ?? '',
         _owner = owner ?? '',
         data = data != null
             ? decodeBase64ToBytes(data)
             : (dataBytes ?? Uint8List(0)),
-        _tags = tags ?? [];
+        _tags = tags ?? [] {
+    if (dataSize != null) {
+      _dataSize = dataSize;
```

Comment:
Why we just assign when it's not `null`? It makes any difference?

Reply:
It's just for maintaining parity with transactions, as there can be transactions without data.